src/usr/local/www/diag_logs_filter.php

```diff
@@ -216,14 +216,14 @@ function build_if_list() {
 		null,
 		'text',
 		$filterfieldsarray['srcip']
-	))->setHelp('Source IP Address');
+	))->setHelp(gettext('Source IP Address'));
 
 	$group->add(new Form_Input(
 		'filterlogentries_destinationipaddress',
 		null,
 		'text',
 		$filterfieldsarray['dstip']
-	))->setHelp('Destination IP Address');
+	))->setHelp(gettext('Destination IP Address'));
 
```

Comment:
> On Dec 3, 2015, at 22:07, Steve Beaver <notifications@github.com> wrote:
> 
> In src/usr/local/www/diag_logs_filter.php:
> 
> >  
> >  	$group->add(new Form_Input(
> >  		'filterlogentries_destinationipaddress',
> >  		null,
> >  		'text',
> >  		$filterfieldsarray['dstip']
> > -	))->setHelp('Destination IP Address');
> > +	))->setHelp(gettext('Destination IP Address'));
> >  
> 
> Both gettext() is applied within the Form_* classes, so it is not necessary to use the here. 
> Other changes in this PR are good, but please revert the gettext()s

gettext() calls must be added direct to the text, not to variables, otherwise xgettext will not be able to get text sentences and produce .pot file.

--
Renato Botelho do Couto
Software Engineer
Netgate, Inc.